package.json

```diff
@@ -305,6 +305,7 @@
     "octokit": "^2.1.0",
     "octokit-plugin-create-pull-request": "^3.11.0",
     "prop-types": "^15.6.1",
+    "querystring": "^0.2.1",
```

Comment:
For now, we're still relying on some grid styles from uswds v1 and foundation-sites, which have a dependency on querystring. When we migrate grid to uswds v3, this dependency will no longer be needed.